package.json

```diff
@@ -4,14 +4,15 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "start": "npm run dev",
     "dev": "nw src/",
     "prod": "nwbuild --platforms win64 --buildDir dist/ src/"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "nw": "^0.51.0-sdk",
+    "nw": "0.51.0-sdk",
```

Comment:
this change means it will actually install the SDK version, so you can use dev tools

Reply:
Oh my god... that's all it took to get dev tools?

Reply:
Yeah, it's annoying, but a quirk of how npm works. You are essentially saying "Find this version or anything slightly newer". and so it finds the non-sdk version as being slightly newer and installs that instead. If you specifically pin it to a `-sdk` version it will install correctly.

If you don't care about version you can also just do `"nw": "sdk"` and it will always install the latest SDK.